### PR TITLE
Initial setup: dal, router/routes/handlers, net configs.

### PR DESCRIPTION
**Pull request by evscott**

### CODE REVIEW

File: src/router/handlers/test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const dal = require('../../dal/dal');
+
+let test = async (req, res) => {
+    dal.testFunction('dal test').then((res) => console.log(res));
```
Comment (gzinck) [inline src/router/handlers/test.js:4]: What is "dal"? Right now, my brain is thinking "Dalhousie University" and that's it 😂

**Review by gzinck — APPROVED** [review]
Your folder structure looks very modular, which is good! Maybe add some extra comments down the road, but overall nice work! 👍

File: src/router/handlers/test.js (hunk 1 above)
Comment (evscott) [reply src/router/handlers/test.js:4]: Database Access Layer! Haha. Your head's in Halifax - _literally_.

**Comment by evscott** [comment]
> Your folder structure looks very modular, which is good! Maybe add some extra comments down the road, but overall nice work! +1

Thanks! Comments are coming! I held off on them with these changes, and will be introducing some with real endpoints/functions/etc.

**Pull request merged**